static/manifest.json

```diff
@@ -4,9 +4,8 @@
   "description": "Learn AMP by Examples",
   "start_url": "https://ampbyexample.com",
   "display": "standalone",
```

Comment:
iOS, FireOS, and possibly some AOSP devices without CTS. 

I'm comfortable deferring given that Safari doesn't honor link rel=manifest, and we're not enabling standalone display on iOS.  If you'd like to do that, I'd suggest opening a separate ticket to track.